src/api/storageManager.ts

```diff
@@ -0,0 +1,32 @@
+import { persist, createJSONStorage, StateStorage } from 'zustand/middleware';
+import { create } from 'zustand';
+import { WHITELIST_KEYS } from '../constants/storageKeys';
```

Comment:
as you done absolute path setup , can we use it here ? '@/constants/storageKeys'; like this ?